src/components/Authentication/Login.jsx

```diff
@@ -30,13 +32,19 @@ const Login = ({ cookies }) => {
   const [password, setPassword] = useState();
   const [showPassword, setShowPassword] = useState(false);
   const [errorMessage, setErrorMessage] = useState();
+  const {
+    register,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'register' is assigned a value but never used.